syntax = "proto3";

message RequestServer{
  string site_id = 1;
  string license_uuid = 2;
}

message Server{
  int64 id = 1;
  string name = 2;
  string server_type = 3;
  string version = 4;
  string host_name= 5;
  string ipaddr = 6;

  string license_data = 7; /* base64 encoded string */
}

message ServerRequest {
  RequestServer server = 1;
  int64 server_id = 2;
}

message ServerResponse {
  Server server = 1;
}

